content/articles/overview-of-developing-serverless-applications-in-c/index.md

```diff
@@ -0,0 +1,144 @@
+
+This article draws attention to developing a serverless application using the [C#](https://docs.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/) programming language. Then later be deployed on popular cloud solutions such as [Amazon AWS](https://aws.amazon.com/) and [Microsoft Azure](https://azure.microsoft.com/en-us/).
+
+### Table of contents
+1. [Introduction](#introduction)
+2. [Developing Azure Functions using C#](#developing-azure-functions-using-c)
+3. [Developing serverless applications on AWS Lambda using C#](#developing-serverless-applications-on-aws-lambda-using-c)
+4. [Using Visual Studio to code in C# for AWS Lambda](#using-visual-studio-to-code-in-c-for-aws-lambda)
+5. [Running the code on AWS Lambda](#running-the-code-on-aws-lambda)
+6. [Conclusion](#conclusion)
+7. [Further reading](#further-reading)
+
+
+### Developing Azure Functions using C#
+#### Prerequisites
+Below are some of the requirements to be met before starting the application development:
+- A developer has to [register an account](https://azure.microsoft.com/en-us/free/) with Microsoft Azure.
+- A developer has to have an account with [Azure Portal](https://portal.azure.com/) to access [Azure Storage](https://docs.microsoft.com/en-us/azure/storage/common/storage-introduction).
+
+Various tools are needed when developing C# applications. The most commonly used are [Visual Studio Code](https://code.visualstudio.com/) and [Visual Studio IDE](https://visualstudio.microsoft.com/vs/). Developers can choose either of the tools depending on their preferences and needs. We will cover both of these tools in detail in the following sections.
+
+#### Using VS Code
+It is a powerful and lightweight code editor that supports different operating systems such as Windows, Linux, and Mac OS. Developers can download either of the versions depending on their platforms.
+
+Azure Tools for VS Code are required to allow the developers to use the commands that interact with Azure functions in the VS Code editor.
+
+To achieve that, we will browse to the VS Code extension manager located at the left side of the window and search for the [Azure Tools extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-azureextensionpack) and install it, as shown in the figure below:
+
+![Extension pack](/engineering-education/overview-of-developing-serverless-applications-in-c/azure-extensions1.png)
+
+Next, will be to sign in to Azure by pressing the keys `CTRL+Shift+P` on the keyboard to launch a Command Palette in the VS Code editor.
+
+Then, a developer will enter the `Azure: Sign In` in the text field displayed on top of VS Code editor.
+
+The developer will click on the results that appear after typing on the text field. VS Code editor will generate a code and display the following steps to be followed in order to complete the set-up.
```

Comment:
...the following steps? Or just `steps`? Following steps seems like you want to list the steps to be followed.